documents-webapp/src/main/webapp/vue-app/attachment/initComponents.js

```diff
@@ -41,7 +44,10 @@ const components = {
   'analytics-table-cell-document-origin-value': AnalyticsTableCellDocumentOriginValue,
   'content-attachment-list': ContentAttachmentList,
   'content-attachment-item': ContentAttachmentItem,
-  'notes-attachment': NotesAttachment
+  'notes-attachment': NotesAttachment,
+  'notes-attachment-list': NotesAttachmentList,
+  'notes-attachment-carousel': NotesAttachmentCarousel,
+  'notes-overview-item': NotesOverviewItem,
```

Comment:
Please rename components

```suggestion
  'notes-attachment-carousel': NotesAttachmentCarousel,
  'notes-attachment-item': NotesAttachmentItem,
```